kod/object/passive/ornobj.kod

```diff
@@ -1286,6 +1403,20 @@ messages:
       {
          send(oDuplicate,@SetNoMoveOn,#value=TRUE);
       }
+      if Send(self, @GetObjectFlags) & FLICKERING_YES
+      {
+         Send(oDuplicate, @SetFlickering, #value=TRUE);
+      }
+      if NOT Send(self, @GetObjectFlags) & LOOK_NO
```

Comment:
I think this needs parentheses.  NOT is higher priority than &.

Reply:
Done.